src/main/flight/gps_rescue.c

```diff
@@ -116,8 +116,9 @@ typedef struct {
     bool isAvailable;
 } rescueState_s;
 
-#define GPS_RESCUE_MAX_YAW_RATE       180  // deg/sec max yaw rate
-#define GPS_RESCUE_RATE_SCALE_DEGREES 45   // Scale the commanded yaw rate when the error is less then this angle
+#define GPS_RESCUE_MAX_YAW_RATE        180  // deg/sec max yaw rate
+#define GPS_RESCUE_RATE_SCALE_DEGREES   45  // Scale the commanded yaw rate when the error is less then this angle
+#define GPS_RESCUE_SLOWDOWN_DISTANCE_M  20  
```

Comment:
Like the other constants, it would be helpful to add a comment describing what the value is for and how it functions for others to more easily understand.

Reply:
Done